victim-identifier/src/agent.spec.ts

```diff
@@ -0,0 +1,279 @@
+      customValue: "",
+    });
+  }
+
+  return Finding.fromObject({
+    name: "Victim Identified - Exploitation Stage",
+    description:
+      "A possible victim has been identified in the exploitation stage of an attack",
+    alertId: "VICTIM-IDENTIFIER-EXPLOITATION-STAGE",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    metadata,
+    labels,
+  });
+};
+
+describe("Victim Identifier bot test suite", () => {
+  const mockGetIdentifiedVictims = jest.fn();
+  const mockVictimIdentifer = {
+    getIdentifiedVictims: mockGetIdentifiedVictims,
+  };
+  let handleTransaction: HandleTransaction = provideHandleTransaction(
+    mockVictimIdentifer as any
+  );
+
+  describe("handleTransaction", () => {
+    it("returns empty findings if there are no victims", async () => {
+      const mockTxEvent = new TestTransactionEvent();
+
+      when(mockGetIdentifiedVictims)
+        .calledWith(mockTxEvent)
+        .mockReturnValue({ preparationStage: {}, exploitationStage: {} });
+
+      const findings = await handleTransaction(mockTxEvent);
+
+      expect(findings).toStrictEqual([]);
+    });
+
+    it("returns a finding if there is a preparation stage victim", async () => {
+      const mockTxEvent = new TestTransactionEvent();
+      const victims = {
+        preparationStage: {
+          "0x0000000000000000000000000000000000011888": {
+            holders: [
+              "0x000000000000000000000000000000000022aabb",
+              "0x0000000000000000000000000000000033aabbcc",
+            ],
+            protocolTwitter: "Victim5678Twitter",
+            protocolUrl: "victim5678.org",
+            tag: "Victim5678",
+            confidence: 1,
+          },
+        },
+        exploitationStage: {},
+      };
+
+      when(mockGetIdentifiedVictims)
+        .calledWith(mockTxEvent)
+        .mockReturnValue(victims);
+
+      const findings = await handleTransaction(mockTxEvent);
+
+      expect(findings).toStrictEqual([
+        testCreatePreparationStageFinding(victims.preparationStage),
+      ]);
+    });
+
+    it("returns a finding if there is an exploitation stage victim", async () => {
+      const mockTxEvent = new TestTransactionEvent();
+      const victims = {
+        preparationStage: {},
+        exploitationStage: {
+          "0x0000000000000000000000000000000000012132": {
+            holders: [],
+            protocolTwitter: "Victim225678Twitter",
+            protocolUrl: "victim225678.org",
+            tag: "Victim225678",
+            confidence: 0.4,
+          },
+        },
+      };
+
+      when(mockGetIdentifiedVictims)
+        .calledWith(mockTxEvent)
+        .mockReturnValue(victims);
+
+      const findings = await handleTransaction(mockTxEvent);
+
+      expect(findings).toStrictEqual([
+        testCreateExploitationStageFinding(victims.exploitationStage),
+      ]);
+    });
+
+    it("returns a findings if there are multiple exploitation stage victims", async () => {
```

Comment:
```suggestion
    it("returns findings if there are multiple exploitation stage victims", async () => {
```
Small fix

Reply:
On this, the error would be the "s" in "finding**s**".